docs/mp/integrations/neo4j.adoc

```diff
@@ -0,0 +1,268 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Neo4j
+:description: Neo4j integration
+:keywords: neo4j
+:feature-name: Neo4j
+:rootdir: {docdir}/../..
+
+include::{rootdir}/includes/mp.adoc[]
+
+== Contents
+
+- <<Overview, Overview>>
+- <<maven-coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+- <<Configuration, Configuration>>
+- <<Examples, Examples>>
+- <<Additional Information, Additional Information>>
+- <<References, References>>
+
+== Overview
+
+Neo4j is a graph database management system developed by Neo4j, Inc. It is an ACID-compliant transactional database with native graph storage and processing. Neo4j is available in a GPL3-licensed open-source “community edition”.
+
+include::{rootdir}/includes/dependencies.adoc[]
+
+[source,xml]
+----
+<dependency>
+   <groupId>io.helidon.integrations.neo4j</groupId>
+   <artifactId>helidon-integrations-neo4j</artifactId>
+</dependency>
+----
+
+== Usage
+
+The support for Neo4j is implemented in drivel level. Just add the dependency, add configuration in `microprofile-config.properties` file and Neo4j driver will be configured by Helidon and can be injected using CDI.
+
+First describe Neo4j connection properties:
+
+[source,properties]
+----
+# Neo4j settings
+neo4j.uri=bolt://localhost:7687
+neo4j.authentication.username=neo4j
+neo4j.authentication.password: secret
+neo4j.pool.metricsEnabled: true
+----
+
+Then just inject the driver:
+
+[source,java]
+----
+@Inject
+public MovieRepository(Driver driver) {
+   this.driver = driver;
+}
+----
+
+The driver can be used according to the link:https://neo4j.com/developer/java/[Neo4j documentation].
+
+== Configuration
+
+include::{rootdir}/config/io_helidon_integrations_neo4j.adoc[leveloffset=+1,tag=config]
+
+== Examples
+
+This example implements a simple Neo4j REST service using MicroProfile. For this example a working Neo4j database is required. Neo4j Movie database is used for this example.
```

Comment:
Maybe add "The" just before "Neo4j Movie database..."

Reply:
Fixed